Themis/Models/Repository.swift

```diff
@@ -80,7 +101,26 @@ class Node: Hashable, ObservableObject {
     var path: String {
         calculatePath().joined(separator: "/")
     }
-
+    
+    private let languageDict: [String: Language] = [
+        "c" : .c,
+        "py" : .python,
```

Comment:
Fun fact: There's also the `.pyw` file extension, which is also associated with Python (for running without a Terminal). We might want to include that as well.
```suggestion
        "py" : .python,
        "pyw": .python,
```